consensus/polybft/fsm.go

```diff
@@ -644,6 +645,10 @@ func validateHeaderFields(parent *types.Header, header *types.Header) error {
 	if header.Number != parent.Number+1 {
 		return fmt.Errorf("invalid number")
 	}
+	// verify time is from the future
+	if header.Timestamp > (uint64(time.Now().UTC().Unix()) + blockTimeDrift) {
+		return fmt.Errorf("block from the future. block time: %d, time drift %d", header.Timestamp, blockTimeDrift)
```

Comment:
```suggestion
		return fmt.Errorf("block from the future. block timestamp: %d, configured block time drift %d seconds", header.Timestamp, blockTimeDrift)
```

Also thinking out loud: should we convert `header.Timestamp` into a proper time format and report it that way in an error message, instead of logging (raw) elapsed seconds? The mindset is that the log message would contain time stamp in date/time format, so it would be maybe easier figuring out what is going on.

Reply:
I agree. I will change that also.